pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
In any case, although beyond the scope of this discussion, level irq can be used anywhere, even if you teach an edge-only system how to do it, such as pic32. It also keeps the code base easier for people to port, and understand. Yes I could have used edge, but actually, on this particular chip, it imposes a penalty--- you can only ACK one IRQ at a time, of the chip gets confused, which means you spin handing one at a time v.s. all at the same time with one shot as it is now... so, level does have its advantages in this case, especially since it does all the work over SPI. If it was PCIX or something, then edge is really the way to go, I agree, but it wouldn't work very well at all when using slow SPI rates. The code as is can work at rates as low as 500KHz SPI... possibly lower, but I haven't tried. Just as long as everything gets done in time to catch the next SOF tick, it will work.